Tests/DefaultsTests/DefaultsTests.swift

```diff
@@ -5,16 +5,27 @@ import XCTest
 import Defaults
 
 let fixtureURL = URL(string: "https://sindresorhus.com")!
+let fixtureFileURL = URL(string: "file://~/icon.png")!
 let fixtureURL2 = URL(string: "https://example.com")!
 
-enum FixtureEnum: String, Codable {
+enum FixtureEnum: String, DefaultsSerializable {
 	case tenMinutes = "10 Minutes"
 	case halfHour = "30 Minutes"
 	case oneHour = "1 Hour"
 }
 
+enum FixtureIntEnum: Int, DefaultsSerializable {
```

Comment:
Can you also add a test for a enum like `enum FixtureIntEnum: Int, Codable, DefaultsSerializable {`?